pygmt/helpers/decorators.py

```diff
@@ -15,13 +15,14 @@
 COMMON_OPTIONS = {
     "R": """\
         region : str or list
-            *Required if this is the first plot command*.
-            ``'xmin/xmax/ymin/ymax[+r][+uunit]'``.
-            Specify the region of interest.""",
+            *xmin/xmax/ymin/ymax*\ [**+r**\ ][**+u**\ *unit*]
```

Comment:
As I mentioned in https://github.com/GenericMappingTools/pygmt/issues/631#issuecomment-764968391, we need to find a good, nice-looking way to separate the `[...*unit*]` argument from the `Specify...` text:

![image](https://user-images.githubusercontent.com/23487320/105418867-b10feb80-5ca2-11eb-8eaf-4bcf6a47d2de.png)

Either add a fullstop/period (.) as suggested below, or a linebreak like in the GMT docs (does `\n` work?), or something else.

```suggestion
            *xmin/xmax/ymin/ymax*\ [**+r**\ ][**+u**\ *unit*].
```
For reference, pygmt v0.2.1 looked like this:

![image](https://user-images.githubusercontent.com/23487320/105419158-2380cb80-5ca3-11eb-9fae-dd440d93c025.png)